docs/README.md

```diff
@@ -0,0 +1,118 @@
+# Overview
+We currently use CircleCI to test each of our integrations. The tests run for **your** integration each time you commit to the content repo. Tests for **every** integration runs nightly. Perhaps the easiest way to create a test playbook is via the Demisto Playbook Editor and remains the preferred way to create one.
```

Comment:
This is internal. Circle will not run for external contributors

Reply:
I'll remove this one from the repo then since it's almost entirely internal